src/Xamarin.CodeAnalysis/Apple/AttributeAnalyzer.cs

```diff
@@ -186,7 +186,12 @@ void MergeSymbolAttributes(OperationAnalysisContext context, ISymbol symbol, INa
             {
                 if (attr.AttributeClass == adviceAttribute && attr.ConstructorArguments.Length > 0)
                 {
-                    AddIssue(context, adviceRule, attr.ConstructorArguments[0].Value?.ToString());
+                    var message = attr.ConstructorArguments[0].Value?.ToString();
+
+                    if (message.Equals("Overriding this method requires a call to the overriden method."))
```

Comment:
This is hacky, but I'm not sure that there's a better way to filter out `[RequiresSuper]` given how this analyzer is currently structured. Open to ideas here.